tests/raftstore/test_multi.rs

```diff
@@ -121,6 +124,26 @@ fn test_multi_lost_majority<T: Simulator>(cluster: &mut Cluster<T>, count: usize
 
 }
 
+fn test_multi_random_restart<T: Simulator>(cluster: &mut Cluster<T>, count: usize) {
+    cluster.bootstrap_region().expect("");
+    cluster.start();
+
+    let (key, value) = (b"a1", b"v1");
+
+    assert_eq!(cluster.get(key), None);
+    cluster.put(key, value);
+    assert_eq!(cluster.get(key), Some(value.to_vec()));
+
+    let mut rng = rand::thread_rng();
+    for _ in 1..100 {
+        let id = 1+rng.gen_range(0, count as u64);
+        cluster.stop_node(id);
+        cluster.run_node(id);
+        sleep_ms(600);
```

Comment:
stop, get, run then sleep.
But I think sleep is not ok, you should use another way to check whether this node joins the cluster or not.